app/src/main/java/com/heapixLearn/discovery/logic/contact/LogicContact.java

```diff
@@ -0,0 +1,55 @@
+package com.heapixLearn.discovery.logic.contact;
+
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.DBStoreContact;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.ServerStoreContact;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.UIContact;
+
+public class LogicContact {
+    DBStoreContact dbContact;
+    ServerStoreContact serverContact;
+    UIContact uiContact;
```

Comment:
@dianaprimshits why do you need these? why don't create fields?

Reply:
@YodaMike Because if I create fields, it will not be a LogicContact, but a contact which ready to display in UI. But I think it's Domain work, not Logic.